examples/generic_connector_configs/README.md

```diff
@@ -1,28 +1,221 @@
 # Generic HTTP Connector Example Configurations
 
+## Table of Contents
+- [Generic HTTP Connector Example Configurations](#generic-http-connector-example-configurations)
+  * [Table of Contents](#table-of-contents)
+- [Sample Configurations](#sample-configurations)
+  * [Elasticsearch API](#elasticsearch-api)
+  * [Github API](#github-api)
+  * [OAuth 2.0 API](#oauth-20-api)
+  * [Slack Web API](#slack-web-api)
+  * [Splunk API](#splunk-api)
+- [Contributing](#contributing)
+
+## Sample Configurations
```

Comment:
What comes below this reads more like an "Introduction". Can we make this section "Introduction" and then below it have a section for "Sample Configurations" that has a sentence or two to explain what's in the section and then jumps in to the sample configs?